components/automate-ui/src/app/entities/automate-settings/automate-settings.model.ts

```diff
@@ -1,73 +1,141 @@
 export class JobSchedulerStatus {
-  running: boolean;
   jobs: IngestJob[];
 
-  constructor(running: boolean, ingestJobs: IngestJob[]) {
-    this.running = running;
+  constructor(ingestJobs: IngestJob[]) {
     this.jobs = ingestJobs;
   }
-
-  getJob(name: string): IngestJob {
-    return this.jobs.find((job: IngestJob) => job.name === name);
-  }
 }
 
 export interface ConfigureSettingsRequest {
   jobs: IngestJob[];
 }
 
 export interface RespJob {
-  running: boolean;
   name: string;
-  every: string;
+  nested_name?: string;
+  disabled: boolean;
+  recurrence: string;
   threshold: string;
-  last_run: Date;
-  next_run: Date;
-  last_elapsed: Date;
-  started_on: Date;
+  purge_policies: any; // needs specific type
+  last_elapsed?: Date;
+  next_due_at?: Date;
+  last_enqueued_at?: Date;
+  last_started_at?: Date;
+  last_ended_at?: Date;
 }
 
 export interface RespJobSchedulerStatus {
-  running: boolean;
-  jobs: RespJob[];
+  infra: {
+    jobs: RespJob[];
+  };
+  compliance: {
+    jobs: RespJob[];
+  };
+  event_feed: {
+    jobs: RespJob[];
+  };
+  services: {
+    jobs: RespJob[];
+  };
```

Comment:
If these top-level properties only have a single child each, wondering if we could compress them to something like this? If the original is what you get from the backend, then there's not much we can do, of course.
```suggestion
  infra_jobs: RespJob[];
  compliance_jobs: RespJob[];
  event_feed_jobs: RespJob[];
  services_jobs: RespJob[];
```
(Same consideration for JobRequestBody)

Reply:
Unfortunately - this is what i'm getting from the backend and matches the API, so I feel it makes more sense to leave it like this.